compiler/il/OMRSymbol.hpp

```diff
@@ -581,6 +584,11 @@ class OMR_EXTENSIBLE Symbol
        * and invokehandle bytecodes.
        */
       DummyResolvedMethod       = 0x00010000,
+      /**
+       * This flag is used to identify the value type default value instance slot address:
+       * &J9Class.flattenedClassCache.defaultValue
```

Comment:
I wonder whether this could be described in a more general way in case any other downstream consumers, besides OpenJ9, might have a use for static default value instances.

Reply:
I'm not sure if it'd be any use to other situations. `StaticDefaultValueInstance` is used to determine to use the relocation record `TR_RelocationRecordStaticDefaultValueInstance` in downstream.  `TR_RelocationRecordStaticDefaultValueInstance` materializes the address `&J9Class.flattenedClassCache.defaultValue`

Reply:
Maybe you could just remove the J9-specific details:  `&J9Class.flattenedClassCache.defaultValue`